### PR TITLE
Changed menu UI and icons

### PR DESCRIPTION
**Pull request by idzkowski-google**

### CODE REVIEW

**Comment by idzkowski-google** [comment]
@googlebot I signed it!

File: CardSample/app/src/main/java/com/example/android/glass/cardsample/menu/GlassMenuItem.java [hunk 1]
```diff
@@ -18,73 +18,73 @@
...
-  }
-
   /**
    * {@link GlassMenuItem} object is constructed by usage of this method.
    *
    * @param icon is a menu icon {@link Drawable} object.
    * @param text is a String with the menu option label.
    */
-  GlassMenuItem(Drawable icon, String text) {
+  GlassMenuItem(int id, Drawable icon, String text) {
```
Comment (jbolinger) [inline CardSample/app/src/main/java/com/example/android/glass/cardsample/menu/GlassMenuItem.java]: Add the doc for the added parameter.

Also double check the visibility on these methods - seems inconsistent but maybe there's a reason?

File: CardSample/app/src/main/res/drawable/ic_add.xml [hunk 2]
```diff
@@ -1,6 +1,5 @@
-<?xml version="1.0" encoding="utf-8"?>
-<!--
- Copyright 2019 Google LLC
+<?xml version="1.0" encoding="utf-8"?><!--
```
Comment (jbolinger) [inline CardSample/app/src/main/res/drawable/ic_add.xml]: comment on wrong line

File: CardSample/app/src/main/res/drawable/ic_delete.xml [hunk 3]
```diff
@@ -1,6 +1,5 @@
-<?xml version="1.0" encoding="utf-8"?>
-<!--
- Copyright 2019 Google LLC
+<?xml version="1.0" encoding="utf-8"?><!--
```
Comment (jbolinger) [inline CardSample/app/src/main/res/drawable/ic_delete.xml]: comment on wrong line

File: CardSample/app/src/main/res/drawable/ic_save.xml [hunk 4]
```diff
@@ -0,0 +1,25 @@
+<?xml version="1.0" encoding="utf-8"?><!--
```
Comment (jbolinger) [inline CardSample/app/src/main/res/drawable/ic_save.xml]: wrong line

File: CardSample/app/src/main/res/drawable/ic_style.xml [hunk 5]
```diff
@@ -0,0 +1,25 @@
+<?xml version="1.0" encoding="utf-8"?><!--
```
Comment (jbolinger) [inline CardSample/app/src/main/res/drawable/ic_style.xml]: line

File: CardSample/app/src/main/java/com/example/android/glass/cardsample/menu/GlassMenuItem.java (hunk 1 above)
Comment (idzkowski-google) [reply CardSample/app/src/main/java/com/example/android/glass/cardsample/menu/GlassMenuItem.java]: All getter methods have the same (public) visibility now.

File: CardSample/app/src/main/res/drawable/ic_add.xml (hunk 2 above)
Comment (idzkowski-google) [reply CardSample/app/src/main/res/drawable/ic_add.xml]: Done

File: CardSample/app/src/main/res/drawable/ic_delete.xml (hunk 3 above)
Comment (idzkowski-google) [reply CardSample/app/src/main/res/drawable/ic_delete.xml]: Done

File: CardSample/app/src/main/res/drawable/ic_save.xml (hunk 4 above)
Comment (idzkowski-google) [reply CardSample/app/src/main/res/drawable/ic_save.xml]: Done

File: CardSample/app/src/main/res/drawable/ic_style.xml (hunk 5 above)
Comment (idzkowski-google) [reply CardSample/app/src/main/res/drawable/ic_style.xml]: Done

File: CardSample/app/src/main/res/drawable/ic_add.xml [hunk 6]
```diff
@@ -1,6 +1,6 @@
 <?xml version="1.0" encoding="utf-8"?>
 <!--
- Copyright 2019 Google LLC
+ Copyright 2020 Google LLC
```
Comment (jbolinger) [inline CardSample/app/src/main/res/drawable/ic_add.xml]: Don't update this for existing files

File: CardSample/app/src/main/res/drawable/ic_delete.xml [hunk 7]
```diff
@@ -1,6 +1,6 @@
 <?xml version="1.0" encoding="utf-8"?>
 <!--
- Copyright 2019 Google LLC
+ Copyright 2020 Google LLC
```
Comment (jbolinger) [inline CardSample/app/src/main/res/drawable/ic_delete.xml]: Don't update this for existing files

File: CardSample/app/src/main/res/drawable/ic_add.xml (hunk 6 above)
Comment (idzkowski-google) [reply CardSample/app/src/main/res/drawable/ic_add.xml]: Done

File: CardSample/app/src/main/res/drawable/ic_delete.xml (hunk 7 above)
Comment (idzkowski-google) [reply CardSample/app/src/main/res/drawable/ic_delete.xml]: Done

**Pull request merged**